### ISSUE TITLE
Using "/" prefix does not work.

### ISSUE THREAD

**Issue by Zach0813** [verzac/grocer-discord-bot#14]
Bot does not accept commands with "/" prefix and does not need to mentioned to use "!" prefix as mentioned in notice.

![image](https://user-images.githubusercontent.com/50189705/162792443-9223812b-fc77-4608-afe0-ddcae9e3276b.png)

**Comment by verzac** [comment]
Hey! This is as expected - the cutoff date for "not mentioning the bot" was at 30 April 2022. However, seems like Discord has pushed the date back to September.

> On September 1, 2022, we will remove verified apps' access to message content if they have not been approved for the Privileged Intent.
https://support-dev.discord.com/hc/en-us/articles/4404772028055-Message-Content-Privileged-Intent-FAQ

The "/" prefix indicates the usage of slash commands. You have to select the slash command first before you can run them; copy-pasting or directly typing it into the chat doesn't work unfortunately (blame Discord for that janky implementation - Slack does it perfectly fine).

![image](https://user-images.githubusercontent.com/15793363/162854398-faee5aea-45ba-4f53-bdcf-59bc4129c105.png)

I'll remove the notice though since the date was pushed back to Sept.

**Comment by verzac** [comment]
(Feel free to make a PR to remove the notice btw and I'd be happy to review - I'm currently busy with other paid gigs)

**Comment by Zach0813** [comment]
Thanks, however the commands do not show up in the available list of commands either when I type “/“.  I have a few other bots such as Dyno, Ticket Tool, and Streamcord that already implemt this fully including typing it out fully.  

I wish I could offer more help to solve this issue as I have a friend who plans on implementing this feature into a bot I have commissioned from him because I don’t know how to do it myself. 

Otherwise loving the work so far!

**Comment by verzac** [comment]
@Zach0813 

TL:DR; re-invite the bot using this [new invite link](https://discord.com/oauth2/authorize?client_id=815120759680532510&permissions=2048&scope=bot%20applications.commands)

Thanks for raising this - turns out the bot needed the `application.commands` scope in its invite link, otherwise you wouldn't be able to access the bot's `/` commands in the server. I didn't catch this at first because they automatically applied the scope for the bot's existing servers last year (including my test server); if the bot joined your server this year, it needs the extra scope.

Updated all the invite links to include this extra scope.

To thank you for your help, I'm going to give you a free "patronage membership" which never expires, which entitles you to 10 max grocery lists and 300 max grocery entries. If you're keen, please do feel free to:
1. Give me your Discord ID / username (e.g. verzac#6377), so that I can apply it to your user. You can reach out to me privately through Discord or post it here.
2. Use `!gropatron register` or `/gropatron register` to register your server.
3. Use `!grohelp` or `/grohelp` to view your new benefits! 

Again, thank you for using GroceryBot and reporting this issue!

**Comment by Zach0813** [comment]
Sorry, I did not get a notification for this since the issue was closed.  
My Discord is Zach329#0410, I very much appreciate the update and membership.  Glad I could help!

**Comment by verzac** [comment]
@Zach0813 no problem! I'm sorry for getting back to you as well since I was super busy the past couple of days

I've added your Discord user to our membership list - try running `!gropatron register` to register your server :)
